### PR TITLE
Added idle animation and changed sprite structure

### PR DESCRIPTION
**Pull request by cmclaudet**
I added the idle animation and made it so the sprite data is separate from the main file.
But the map seems to have changed... The NPC characters are still there but the other background stuff is GONE. I dunno what happened. Maybe check for differences between the maps?

### CODE REVIEW

File: src/main.lua [hunk 1]
```diff
@@ -1,112 +1,93 @@
...
+
+	--player table allows easy access for player attributes and controls
+	player = {
+		x = playerX,
+		y = playerY,
+		speed = playerspeed,
+		char = booboo,
+		down = "down",
+		right = 'right',
+	  left = 'left',
```
Comment (emilyclaudet) [inline src/main.lua]: This line needs to be aligned for neatness :P

**Comment by emilyclaudet** [comment]
This is AMAZING Atherine. Love it. Feel free to merge it :shipit: 

The reason the map detail is gone is due to a change in the resolution we made in the conf.lua file. We changed it from 1024x728 to 800x600 which meant that all the other background stuff isn't visible (really dumb I know). Not bothered to fix that atm as we're going to change the map anyway.

Definitely thinking we need to scale up and display the map + characters at 2x the size. 

Made an issue on the scaling here:
https://github.com/emilyclaudet/family-game/issues/11

File: src/main.lua [hunk 2]
```diff
@@ -1,112 +1,93 @@
 local STI = require "libs.STI"
-local avatar
 local playerX = 200
 local playerY = 200
-local speed = 200
+local playerspeed = 200
```
Comment (emilyclaudet) [inline src/main.lua]: We may want to slow the characters down a bit... :P

File: src/main.lua (hunk 1 above)
Comment (cmclaudet) [inline src/main.lua]: lol yes I will fix that, don't know how i missed it..

File: src/main.lua (hunk 2 above)
Comment (cmclaudet) [inline src/main.lua]: Ok thanks I'll merge it later today after changing the speed and alignment stuff. I'm glad you think it's good!!

**Pull request merged**